### ISSUE TITLE
Unhandled exception when not having PROFILES path setup

### ISSUE THREAD

**Issue by liuloppan** [OpenSpace/OpenSpace#1408]
We have the use case of having our own openspace.cfg setup and noticed that the new profiles feature throws unhandled exceptions when the PROFILES-path is not set up.

**Comment by alexanderbock** [comment]
```
    if (global::configuration->profile.empty()) {
        LFATAL("Cannot launch with an empty profile");
        exit(EXIT_FAILURE);
    }
```
escapes when not setting a profile in the `openspace.cfg`

**Comment by alexanderbock** [comment]
Fixed as of 6c088a56
